### DISCUSSION TITLE
How to unbind a keymap?

### DISCUSSION THREAD

**Discussion by vunhatchuong** [jeffreytse/zsh-vi-mode#290] [Q&A]
I want to unbind `^f` since I have my own mapping.

I put my keymap after the plugin, tried `lazy=false`, `initialization=sourcing`, `zvm_bindkey viins "^F" ""`. All doesn't work.

**Comment by PILIHU2022** [comment]
> I want to unbind `^f` since I have my own mapping.
> 
> I put my keymap after the plugin, tried `lazy=false`, `initialization=sourcing`, `zvm_bindkey viins "^F" ""`. All doesn't work.

# So if you put your own keybind before this plugin, it works well?

## I am also having the same issue like you.  I edited the code. If you use `zi`, just go to `zi/plugins` and find the dir with `zsh-vi-mode`, and open the `zsh-vi-mode.zsh`, then find `^F`, just replace that with the other keybind.

## And you can also create a GitHub repo to fork this repo. And repeat the steps above(just edit the file). Then input your own repo link.

## And you can also delete the code

# Best wish!
# Not very good in English.
